src/shims/windows/sync.rs

```diff
@@ -7,10 +7,6 @@ use crate::concurrency::sync::{CondvarLock, RwLockMode};
 use crate::concurrency::thread::MachineCallback;
 use crate::*;
 
-const SRWLOCK_ID_OFFSET: u64 = 0;
-const INIT_ONCE_ID_OFFSET: u64 = 0;
-const CONDVAR_ID_OFFSET: u64 = 0;
```

Comment:
Please either keep these constants, or add comments similar to [these](https://github.com/rust-lang/miri/blob/4ca3e276448ee5555f419d2a006ace4a60988388/src/shims/unix/sync.rs#L50-L57) to explain the data layout. Just a magic '0' without context is too confusing.